README.md

```diff
@@ -40,3 +40,37 @@ These packages will be produced as part of the standard build task.
  ```
   -$ ./gradlew build
  ```
+
+## Adding Root CA Certificates
+Self signing certificates can be very useful for local testing & development with SSL enabled. 
+If openEQUELLA is configured to use SSL with such a certificate, then the admin-console-package will fail to log in to the institution because of this error: 
+
+```
+Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: 
+PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: 
+unable to find valid certification path to requested target
+```
+
+This is because the Java Keystore used by the admin console package does not recognize the Root CA certificate used to sign openEQUELLA's SSL certificate.
+The admin console package uses its own copy of the JRE, in the `jdk8u242-b08-jre` folder. 
+The keystore we need to update is within this folder, at the path `jdk8u242-b08-jre/lib/security/cacerts`.
+
+Java comes with a command line tool which you can use for updating these keystores, called `keytool`. This should work in Mac, Linux and Windows.
```

Comment:
Could be good to highlight the _bundled_ java comes with `keytool` and that it should be used over any other ones installed on the system to ensure compatibility. Then handy to provide the location of it: `jdk8u242-b08-jre/bin`

Reply:
Good call - the one I used was probably not the bundled one and it happened to work. Added to the readme.